indico/modules/events/editing/operations.py

```diff
@@ -89,12 +89,13 @@ def create_new_editable(contrib, type_, submitter, files, initial_state=InitialR
     return editable
 
 
-def delete_editable(editable):
+def delete_editable(editable, soft=True):
     db.session.expire(editable)
-    for revision in editable.revisions:
-        for ef in revision.files:
-            ef.file.claimed = False
-    db.session.delete(editable)
```

Comment:
In case of `not soft` this needs to stay.

Reply:
but this is correct, right? (the added code is below)

Reply:
aah, i see what you mean